spec/system/submissions/review_spec.rb

```diff
@@ -1954,4 +1954,55 @@
       expect(page).to have_text("A new report description")
     end
   end
+
+  context "When a milestone submission is ungraded after certificate is issued" do
```

Comment:
In this modification, we are not dealing with anything related to certificates. So this could just be:

_when a milestone submission is ungraded after the course is marked complete for a student_